app/persistence/fabric/CRUDService.ts

```diff
@@ -82,31 +82,51 @@ export class CRUDService {
 	 * @param {*} txid
 	 * @param {*} from
 	 * @param {*} to
+	 * @param {*} page
+	 * @param {*} size
 	 * @param {*} orgs
 	 * @returns
 	 * @memberof CRUDService
 	 */
-	getTxList(
+	async getTxList(
 		network_name: any,
 		channel_genesis_hash: any,
 		blockNum: any,
 		txid: any,
 		from: any,
 		to: any,
-		orgs: string
+		orgs: string,
+		page: any,
+		size: any
```

Comment:
[nitpick]: type will be number.

Reply:
updates the code as suggested.

Reply:
Please define the types for all the parameters.